app/tests/test_user.py

```diff
@@ -0,0 +1,49 @@
+from ..models import User
+from ..serializers import UserSerializer
+from .base import BaseTestCase
+
+
+class UserTestCase(BaseTestCase):
+
+    def test_get_all_users(self):
+        response = self.client.get('/users/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_retrieve_valid_single_user(self):
+        response = self.client.get('/users/{}/'.format(self.user.pk))
```

Comment:
You can use `f strings`, I believe. ` response = self.client.get(f'/users/{self.user.pk}/')`